tests/test_indexer.py

```diff
@@ -330,6 +341,51 @@ def get_search_results(cls, query, expected_hit_count):
             else:
                 time.sleep(0.5)
 
+    def create_sample_bundle_created_event(self, bundle_key):
+        return self.create_bundle_created_event(bundle_key, self.test_bucket)
+
+    def create_bundle_created_event(self, bundle_key, bucket_name):
+        raise NotImplemented()
+
+    def process_new_indexable_object(self, event, logger):
+        raise NotImplemented()
+
+
+class TestAWSIndexer(TestIndexerBase, unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        cls.replica = "aws"
```

Comment:
I think it's a suboptimal idiom to communicate to a superclass method via a variable.

I would prefer that TestIndexerBase have a method called `TestIndexerSetup` that takes a `replica` argument.  Then the method is responsible for setting cls.replica.

Reply:
modifed to only pass in replica